contracts/sol6/KyberNetwork.sol

```diff
@@ -1112,12 +1112,16 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
             }
             totalBps += reservesData.splitsBps[i];
 
-            destAmount += calcDstQty(
+            uint256 destQty = calcDstQty(
```

Comment:
this fix probably doesn't belong here.

Reply:
yes, already added in description of this PR, some fixes here just to be able running the fuzz tests